src/nginxconfig/templates/setup_sections/ssl_cert.vue

```diff
@@ -159,13 +183,54 @@ THE SOFTWARE.
                         ].filter(x => x !== null).join(' ')
                     )).join('\n');
             },
+            acmeShCmds() {
+                return this.$props.data.domains
+                    .filter(domain => domain.https.certType.computed === 'letsEncrypt')
+                    .map(domain => (
+                        [
+                            'acme.sh --issue',
+                            `-d ${domain.server.domain.computed}`,
+                            domain.server.wwwSubdomain.computed ? `-d www.${domain.server.domain.computed}` : null,
+                            domain.server.cdnSubdomain.computed ? `-d cdn.${domain.server.domain.computed}` : null,
+                            `-w ${domain.server.path.computed}${domain.server.documentRoot.computed}`,
+                        ].filter(x => x !== null).join(' ')
+                    )).join('\n');
+            },
+            acmeCopy() {
+                return this.$props.data.domains
+                    .filter(domain => domain.https.certType.computed === 'letsEncrypt')
+                    .map(domain => (
+                        [
+                            `cp -R /root/.acme.sh/${domain.server.domain.computed}`,
+                            domain.server.wwwSubdomain.computed ? `-d www.${domain.server.domain.computed}` : null,
+                            domain.server.cdnSubdomain.computed ? `-d cdn.${domain.server.domain.computed}` : null,
+                            this.$props.data.global.https.letsEncryptCertRoot.computed.replace(/\/+$/, '') + '/' + domain.server.domain.computed,
+                        ].filter(x => x !== null).join(' ')
+                    )).join('\n');
+            },
+            acmeReload() {
+                const letEncryptPath = this.$props.data.global.https.letsEncryptCertRoot.computed.replace(/\/+$/, '') + '/';
+                return this.$props.data.domains
+                    .filter(domain => domain.https.certType.computed === 'letsEncrypt')
+                    .map(domain => (
+                        [
+                            'acme.sh --install cert',
+                            `-d ${domain.server.domain.computed}`,
+                            domain.server.wwwSubdomain.computed ? `-d www.${domain.server.domain.computed}` : null,
+                            domain.server.cdnSubdomain.computed ? `-d cdn.${domain.server.domain.computed}` : null,
+                            '--keyfile ' + letEncryptPath + domain.server.domain.computed + '/' + domain.server.domain.computed +'.key',
+                            '--fullchain-file ' + letEncryptPath + domain.server.domain.computed + '/' + 'fullchain.cer',
+                            '--reload cmd \'systemctl restart nginx.service\'',
+                        ].filter(x => x !== null).join(' ')
+                    )).join('\n');
+            },
```

Comment:
I do not believe a reload step is needed here, the updated copy step will handle this as part of that command